code/modules/mining/ores_coins.dm

```diff
@@ -293,6 +293,14 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
 		return
 	qdel(src)
 
+//plasa magmite functions like an ore in the way it's mined, but we don't want to be redeeming this. Since Gibtonite is stuck with other ores it's okay to put it here.
+/obj/item/magmite
+	name = "plasma magmite"
+	desc = "A chunk of plasma magmite, crystallized deep under lavaland's surface. Its strength seems to fluctuate depending on the distance to the planet."
+	icon = 'icons/obj/mining.dmi'
+	icon_state = "Magmite ore"
```

Comment:
no

Reply:
what do u mean no

Reply:
lowercase and seperated by underscore
magmite_ore

Reply:
this matches the other ore icons, so no